lib/pushmi_pullyu.rb

```diff
@@ -39,7 +39,9 @@ module PushmiPullyu
       auth_url: 'http://localhost:8080/auth/v1.0',
       project_name: 'demo',
       project_domain_name: 'default',
-      container: 'ERA'
+      container: 'ERA',
+      auth_version: 'v1',
```

Comment:
Having v3 as the default was my initial approach, but considered having the current version as to not mess with current behaviour in case we want to go back to it with the transition.

Having v3 as default makes sense now that more people took a look at it.

Will make the change.